src/test/java/codeu/model/store/persistence/PersistentStorageAgentTest.java

```diff
@@ -44,6 +44,14 @@ public void testLoadMessages() throws PersistentDataStoreException {
     Mockito.verify(mockPersistentDataStore).loadMessages();
   }
 
+  @Test
+  public void testUpdateThrough() throws PersistentDataStoreException {
+    User user = new User(UUID.randomUUID(), "test_username", "test_password", Instant.now(), "https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?resize=256%2C256&quality=100");
+    String photo_url = "https://i1.wp.com/www.winhelponline.com/blog/wp-content/uploads/2017/12/user.png?resize=256%2C256&quality=100";
```

Comment:
Can you save this URL as a constant/variable and use it in this line and the above?